### PR TITLE
fix process styles when stylesheets get updated

### PR DESCRIPTION
**Pull request by DevSide**
Some css-in-js librairies, like styled-components, which rewrite its stylesheets on page load, it is necessary to process again the styles on **CSSStyleSheet.insertRule**.

(for css-in-js user: it fixes css hover, urls and so on)

### CODE REVIEW

File: src/client/sandbox/style.js [hunk 1]
```diff
@@ -225,6 +225,11 @@ export default class StyleSandbox extends SandboxBase {
             }
         });
 
+        window.CSSStyleSheet.prototype.insertRule = function (value, index) {
+            value = styleProcessor.process(value, getProxyUrl);
+            nativeMethods.styleSheetsInsertRule.call(this, value, index);
```
Comment (LavrovArtem) [inline src/client/sandbox/style.js]: Please add the blank line before the native method call and return the result of the native method execution.
Also, the returned value should be tested :)
https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/insertRule#Return_value

**Comment by DevSide** [comment]
Without SSR, i guess we also need to listen to <style> tag injection. I don't know if the hammerhead already does it. I will check this.

Anyone know why the tests fail ? CSSStyleSheet is maybe not available on some browsers.

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by DevSide** [comment]
Perfect ! Thanks @LavrovArtem

**Pull request merged**